src/rust/engine/process_execution/src/lib.rs

```diff
@@ -442,8 +442,7 @@ impl CommandRunner for BoundedCommandRunner {
       .unwrap_or_else(|| "<Unnamed process>".to_string());
     let outer_metadata = WorkunitMetadata {
       desc: Some(desc.clone()),
-      level: Level::Info,
-      display: false,
+      level: Level::Trace,
```

Comment:
Debug here probably wouldn't hurt... we run few enough processes that knowing when one is waiting is good.